coding_style/README.adoc

```diff
@@ -82,7 +84,9 @@ Examples:: there are multiple ways to avoid long lines but the most generic one
       Duis a neque fermentum nulla imperdiet blandit.
 ----
 +
-TIP: use the sign `>-` if it is important that the last line return code doesn't become part of the string (e.g. when defining a string variable).
+TIP: Always use the sign `>-` instead of `>` unless you are absolutely sure the trailing newline is not significant.
```

Comment:
```suggestion
TIP: Always use the sign `>-` instead of `>` unless you are absolutely sure the trailing newline is not significant.
The sing `>` adds a newline character to the last line, effecively turning `non nunc.` at the end of the example string above into  `"non nunc.\n"`,`>-` doesn't add the newline character.
```